src/i18n/dim_en.json

```diff
@@ -294,6 +303,8 @@
       "ShowAnimationsHelp": "Will show the animated new item overlay on new items. Turning this off can save CPU cycles.",
       "ShowElemental": "Show elemental damage icons on weapons",
       "ShowElementalHelp": "Show elemental damage icons on weapons.",
+      "AllowIdPostToDtr": "Allow potentially identifying item information to be posted to Destiny Tracker.",
+      "AllowIdPostToDtrHelp": "Allow DIM to post identifying item data to DTR.",
```

Comment:
Might also be an idea to mention which information can be send to Destiny Tracker.

- **InstanceId** _When getting Weapon Reviews, Submitting a Review. A unique ID specific to the player's version._
- **MembershipId** _When submitting a Review, Unique ID identifying the players account._
- **DisplayName** _When submitting a Review, Name of the players account (gamertag, PSN ID)._